poshc2/server/Payloads.py

```diff
@@ -214,15 +214,14 @@ def PatchBytes(self, filename, dll, offset, payloadtype, name=""):
 
         elif payloadtype == PayloadType.Sharp:
             srcfilename = "%s%s%s" % (self.BaseDirectory, name, "dropper_cs.exe")
-            with open(srcfilename, "rb") as b:
-                dllbase64  = base64.b64encode(b.read()).decode("utf-8")
+            dllbase64 = open(srcfilename, "rb").read()
```

Comment:
You want to use the `with open() as srcfile` format here as the `with` statement auto tears down resources when they're out of scope, which can help avoid issues where handles get left open or more than one thing is trying to lock a file at a time because the stream wasn't closed etc.

Reply:
Done